great-idea-website/css/index.css

```diff
@@ -68,81 +68,72 @@ h1, h2, h3, h4, h5 {
 .main-content-area {
 	width: 900px;
 	display: block;
-	margin: auto;
-	margin-top: 20px;
+	margin: 0 auto;
 }
 
 /* Main Navigation Styling */
 
 .main-nav {
-	margin: auto;
-	/*text-align: center;*/
+	margin: 20px auto;
 	width: 900px;
-	height: 100px;
-	position: relative;
+	height: 50px;
+	text-align: center;
 }
 
-nav div {
+nav {
 	display: inline-block;
-	/*height: 60px;*/
-	margin: 25px 50px 25px 0px;
+	vertical-align: middle;
 }
 
 nav a {
-	display: inline-block;
-	padding: 30px;
 	text-decoration: none;
-	color: gray;
-	font-size: 15px;
-
-}
-
-nav a:first-child {
-	padding-left: 0;
+	padding: 0 25px;
+	color: grey;
 }
 
-.logo {
-	position: absolute;
+.main-nav img {
+	display: inline-block;
+	vertical-align: middle;
+	margin-left: 50px;
 }
 
-/* Index.html styling */
+/* Header styling */
 
 header {
-	position: relative;
-	height: 450px;
-	margin: 50px 0px;
+	border-bottom: 2px solid black;
+	margin: 50px 0;
 	padding-bottom: 50px;
-	border-bottom: 1px solid black;
+}
+header div {
+	display: inline-block;
+	vertical-align: middle;
```

Comment:
great job getting the vertical-align : middle to work.  I've found that vertical-align doesn't seem to always work when I try to use it the way I want.